docs/quickstart.ipynb

```diff
@@ -6,45 +6,151 @@
    "source": [
```

Comment:
<p>Trying out a review from the review nb (I failed with this the other day) - I think this is my favorite new section =) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/s3alfisc/pyfixest/pull/436/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>